build/script.js

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
```

Comment:
@jbarget I've renamed `script.js` to `iife_to_init.js` to better describe what the file does (but it's only temporary to get it working)